manifests/e/Elastic/Packetbeat/8.7.1/Elastic.Packetbeat.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Elastic.Packetbeat
+PackageVersion: 8.7.1
+PackageLocale: en-US
+Publisher: Elastic
+PackageName: Packetbeat

```

```suggestion
PackageName: Beats packetbeat
```
to match Control Panel.